addons/website_sale_comparison/views/website_sale_comparison_template.xml

```diff
@@ -127,7 +127,7 @@
                                                 <input name="product_id" t-att-value="product.id" type="hidden"
                                                        t-att-data-product-tracking-info="json.dumps(request.env['product.template'].get_google_analytics_data(combination_info))"/>
                                                 <a t-if="combination_info['prevent_zero_price_sale']" t-att-href="website.contact_us_button_url" class="btn btn-primary btn_cta">Contact Us</a>
-                                                <a t-else="" role="button" class="btn btn-primary d-block a-submit" href="#">
+                                                <a t-else="" role="button" class="btn btn-primary a-submit" href="#">
```

Comment:
?

Reply:
UX issue. Just to have the same button size